packages/presentation/src/components/AssigneePopup.svelte

```diff
@@ -214,11 +214,13 @@
               <!--Category for first item-->
               <div class="category-box">
                 <div class="flex flex-grow overflow-label">
-                  <span class="fs-medium flex-center gap-2 mt-2 mb-2 ml-2">
+                  <span class="fs-medium flex-center gap-1 mt-2 mb-2 ml-2">
                     {#if cl.icon}
                       <Icon icon={cl.icon} size={'small'} />
                     {/if}
-                    <Label label={getCategorytitle(category)} />
+                    <div>
```

Comment:
I'm not sure gap-1 and gap-2 are working properly, better to add ml-1 to this div.